mteb/tasks/Retrieval/code/CodeSearchNetRetrieval.py

```diff
@@ -38,7 +38,53 @@ class CodeSearchNetRetrieval(MultilingualTask, AbsTaskRetrieval):
         n_samples={
             _EVAL_SPLIT: 1000,
         },
-        avg_character_length={"test": 1196.4609},
+        avg_character_length={
```

Comment:
Just need to fix the pydantic validation errors.

Reply:
Looks like you fixed the pydantic, thank you! I'll update the docstring now